### PR TITLE
add env humanoidly

### PR DESCRIPTION
**Pull request by zdx3578**
can train a humanoid lying to  learn  standup

### CODE REVIEW

File: gym/monitoring/stats_recorder.py [hunk 1]
```diff
@@ -68,5 +68,5 @@ def flush(self):
                 'initial_reset_timestamp': self.initial_reset_timestamp,
                 'timestamps': self.timestamps,
                 'episode_lengths': self.episode_lengths,
-                'episode_rewards': self.episode_rewards,
+                'episode_rewards': str(self.episode_rewards),
```
Comment (jietang) [inline gym/monitoring/stats_recorder.py]: what's the purpose of this change?

**Comment by jietang** [comment]
Neat, thanks for adding this. I'd love to merge it in.

I feel like Humanoidly isn't a descriptive enough name - what do you think about renaming it something like HumanoidStandup?

**Comment by zdx3578** [comment]
ok rename push
